wgpu-hal/src/dx12/device.rs

```diff
@@ -1358,8 +1358,8 @@ impl crate::Device<super::Api> for super::Device {
             DepthBias: bias.constant,
             DepthBiasClamp: bias.clamp,
             SlopeScaledDepthBias: bias.slope_scale,
-            DepthClipEnable: if desc.primitive.unclipped_depth { 0 } else { 1 },
-            MultisampleEnable: if desc.multisample.count > 1 { 1 } else { 0 },
+            DepthClipEnable: desc.primitive.unclipped_depth.into(),
```

Comment:
This seems to now be inverted.

Reply:
D'oh, good catch. Fixed with d210a2b13779df16a581097e6adccfe3169adb21.